platform/darwin/src/MGLGeometry.h

```diff
@@ -45,6 +45,18 @@ typedef struct MGLCoordinateBounds {
     CLLocationCoordinate2D ne;
 } MGLCoordinateBounds;
 
+typedef struct MGLCoordinateQuad {
+    /** Coordinate at the northwest corner. */
+    CLLocationCoordinate2D tl;
+    /** Coordinate at the northeast corner. */
+    CLLocationCoordinate2D tr;
```

Comment:
Is it top-right or northeast? Can the developer line up the top-right corner with the southwest geographic extent to rotate the image?